ui/imports/shared/controls/chat/ProfileHeader.qml

```diff
@@ -284,6 +284,7 @@ Item {
         id: editImageMenuComponent
 
         StatusMenu {
+            onClosed: destroy()
```

Comment:
why not?

Reply:
Why not what? :)

Reply:
:man_facepalming: Sorry, I understood you removed that line, but you added actually